tests/draft2020-12/additionalProperties.json

```diff
@@ -2,6 +2,7 @@
     {
         "description":
             "additionalProperties being false does not allow other properties",
+        "specification": [ { "core":"10.3.2.3", "quote": "The value of \"additionalProperties\" MUST be a valid JSON Schema. Boolean \"false\" forbids everything." } ],
```

Comment:
A good follow up (to this ticket, perhaps even before we go much further) would be to make CI annotate pull requests with links to the section.

It'd be a lot easier to check whether this was correct if something were annotating the review with a link to this section.

Reply:
So, for each version of the draft, we should add different "baseURI's" right which should append and check whether the the redirection to the section is failing or not? 
And Ci script should be written separately for that.. I am new to this, may take time. But i'll do it

Reply:
Probably custom logic for each specification which knows what the right URL is given what we have put in the corresponding field, yes, which the simplest version is just "a base URI".

But exactly right. Some CI script for that, which annotates PRs, and warns for broken links.

Can you perhaps immediately open an issue at least so we don't forget about it? Then yeah if you give it a shot that'd be amazing.